test/cypress/integration/consumer-tools/youth-financial-education/survey-errors.cy.js

```diff
@@ -2,20 +2,21 @@ import { TdpSurveyHelpers } from './survey-helpers.cy.js';
 
 const survey = new TdpSurveyHelpers();
 
-describe('Youth Financial Education Survey: Errors', () => {
-  /**
-   *
-   */
-  function refreshErrors() {
-    cy.window().then((win) => win.sessionStorage.clear());
-    survey.open('3-5/p1');
-    survey.selectAnswers([0, 0, null, 0, 0, null]);
-    survey.clickNext();
-  }
+/**
+ * Reset test state.
+ */
+function refreshErrors() {
+  cy.window().then((win) => win.sessionStorage.clear());
+  survey.open('3-5/p1');
+  survey.selectAnswers([0, 0, null, 0, 0, null]);
+  survey.clickNext();
+}
 
+describe('Youth Financial Education Survey: Errors', () => {
   it('jumps to errors at top', () => {
     refreshErrors();
     cy.wait(1200);
+    cy.get('main h1').isScrolledTo();
     cy.window().then((win) => {
       expect(win.scrollY).lessThan(400);
     });
```

Comment:
Now that we're checking to see if a certain element is in view, I wonder if the check for the correct `scrollY` coordinates is redundant? From what I can tell, the intention of the test is "make sure the right part of the page is visible after taking this action"; seems like that would be satisfied by checking that all the right elements pass the `isScrolledTo` assertion. I might be misunderstanding what's actually being checked there, though.

Reply:
Works for me, I think you are right. Changed in https://github.com/cfpb/consumerfinance.gov/pull/7424/commits/597c32f8ade337145d4f10b4b0cc9eab3d0475f2